common/src/app/common/pages/changes_builder.cljc

```diff
@@ -224,10 +253,13 @@
   ([changes ids update-fn]
    (update-shapes changes ids update-fn nil))
 
-  ([changes ids update-fn {:keys [attrs ignore-geometry?] :or {attrs nil ignore-geometry? false}}]
-   (assert-page-id changes)
+  ([changes ids update-fn {:keys [attrs ignore-geometry? ignore-touched]
+                           :or {attrs nil ignore-geometry? false ignore-touched false}}]
```

Comment:
yo don't need `nil` as default value, because it is already will be `nil`.

Reply:
the same for the rest, of falses

Reply:
Done!

It was for documentation. But you are true, actually i'ts not needed.